recipes/pyjpegls/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+    version: 1.5.1
+    python_min: 3.10
+package:
+    name: pyjpegls
+    version: ${{ version }}
+
+source:
+    - url: https://pypi.org/packages/source/p/pyjpegls/pyjpegls-${{ version }}.tar.gz
```

Comment:
I'm not sure if this is allowed, but using a dash here is at least inconsistent (and I didn't see dashes in the V1 docs))